src/materials/ShadowMaterial.js

```diff
@@ -35,6 +36,11 @@ function ShadowMaterial() {
 		}
 	} );
 
+	this.setValues( parameters );
```

Comment:
I just wanted to make sure that `transparent` and `lights` always set to `true`. But yes, it's more understandable when the user values don't change.